src/quacc/utils/files.py

```diff
@@ -60,13 +62,51 @@ def copy_decompress(source_files: list[str | Path], destination: str | Path) ->
     -------
     None
     """
+
+    # Proposal to make the copy file more flexible
+    # i.e add glob support. (Dangerous maybe?)
+    # Just testing for now
```

Comment:
Looks like this comment can be removed. For more flexible glob support, let's try that in a follow-up PR since it's independent of QE.